lib/req-res/status.js

```diff
@@ -11,7 +17,12 @@ module.exports = function ( respond ) {
 		},
 
 		'error' : function ( data, errorData, code ) {
-			console.log( 'Error:', data, errorData, code );
+
+			log.error( data, {
```

Comment:
I don't be believe we have access to the message that initiated the request., so I can't think of anything other than this at the moment.